package.json

```diff
@@ -18,8 +18,8 @@
     "test": "npm run test-base && npm run test-utilities",
     "test-experimental": "npm run test-recurrent && npm run test-utilities",
     "dist": "babel src --out-dir dist --source-maps",
-    "browser": "browserify ./index.js -p licensify -o browser.js",
-    "browser-min": "browserify ./index.js -p licensify -g uglifyify -o browser.min.js",
+    "browser": "browserify ./index.js -p licensify -o browser.js -s brain",
```

Comment:
https://github.com/browserify/browserify#usage
so `-s` gives us `brain` as the exported value.  I believe this is the more appropriate means of achieving an output.